src/redux/slices/debug-options-slice.ts

```diff
@@ -0,0 +1,89 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+import {
+  DebugOptions,
+  TileColoredBy,
+  BoundingVolumeColoredBy,
+  BoundingVolumeType,
+} from "../../types";
+import { RootState } from "../store";
+
+// Define a type for the slice state
+export interface DebugOptionsState {
+  minimap: boolean;
+  minimapViewport: boolean;
+  boundingVolume: boolean;
+  tileColorMode: TileColoredBy;
+  boundingVolumeColorMode: BoundingVolumeColoredBy;
+  boundingVolumeType: BoundingVolumeType;
+  pickable: boolean;
+  loadTiles: boolean;
+  showUVDebugTexture: boolean;
+  wireframe: boolean;
+}
+const initialState: DebugOptionsState = {
+  // Show minimap
```

Comment:
Comment should be for the type rather then for initialState values. Use TSDoc comments /** */